docs/pages/clients/getting-started.md

```diff
@@ -32,28 +32,78 @@ The Development Client uses deep links to open projects from the QR code. If you
+
+<Tab >
+
+<InstallSection packageName="expo-dev-client" cmd={["eas build --profile development --platform ios"]} hideBareInstructions />
+
+</Tab>
+
+<Tab >
+
+<InstallSection packageName="expo-dev-client" cmd={["eas build --profile development --platform android"]} hideBareInstructions />
+
+</Tab>
+
+</Tabs>
+
+and installing the resulting build on your device.
+
+### Locally
+
+If you are comfortable setting up Xcode, Android Studio, and related dependencies, you can build and distribute your app the same as any other iOS or Android application (after running `expo prebuild` to generate the native projects if you are using config plugins).
+
+The `expo run` commands will run a new build, install it in your emulated device, and launch you into your application.
+
+<Tabs tabs={["For iOS Simulator (MacOS Only)", "For Android Emulator"]}>
+
+<Tab >
 
 <InstallSection packageName="expo-dev-client" cmd={["expo run:ios"]} hideBareInstructions />
 
-or your Android emulator
+</Tab>
+
+<Tab >
 
 <InstallSection packageName="expo-dev-client" cmd={["expo run:android"]} hideBareInstructions />
 
-If you are eager to install your project on a physical device, we recommend using [EAS Build](eas-build.md) for the smoothest experience, but you can build and distribute the same as any other React Native application. Once it's installed, you're ready to start developing by running:
+</Tab>
+
+</Tabs>
+
+
+## Developing your application
+
+As you can see, creating a new native build from scratch takes long enough that you'll be tempted to switch tasks and lose your focus.
+
+Now that you have a custom client for you project installed on your device, though, you won't have to wait for the native build process again until you change the native runtime!
+
+Instead, you can start developing in a fraction of the time by running:
 
 <InstallSection packageName="expo-dev-client" cmd={["expo start --dev-client"]} hideBareInstructions />
 
-## Loading your application
+and scanning the resulting QR code with your system camera or QR code reader (if you want to develop against a physical device)
+
+or pressing the A or I keys (to open the app in your Android or iPhone emulator respectively).
+
+Now make some changes to your application code and see them reflected on your device!
+
+### The launcher screen
 
-When you first launch your application, you will see a screen that looks like this:
+If you launch your custom development client from your device's Home Screen, you will see your launcher screen, which looks like this:
 
 <ImageSpotlight alt="The launcher screen of the Development Client" src="/static/images/dev-client-launcher.png" style={{ maxWidth: 225}} />
 
-If a bundler is available on your local network, or you've signed in to your Expo account, you can connect to it directly from this screen.
-Otherwise, you can connect by scanning the QR code displayed by Expo CLI.
```

Comment:
I think it's worth leaving this in, maybe just included in the previous paragraph.